internal/validations/clusterlogforwarder/validate_clusterlogforwarderspec.go

```diff
@@ -201,10 +201,8 @@ func verifyInputs(spec *loggingv1.ClusterLogForwarderSpec, status *loggingv1.Clu
 			badInput("ReceiverSpec must define an HTTP receiver")
 		case isHTTPReceiver(input) && input.Receiver.HTTP.Format != loggingv1.FormatKubeAPIAudit:
 			badInput("invalid format specified for HTTP receiver")
-		case isHTTPReceiver(input) && isInvalidPort(input.Receiver.HTTP.ReceiverPort.Port):
+		case isHTTPReceiver(input) && isInvalidPort(input.Receiver.HTTP.Port):
```

Comment:
is it going to be a problem to allow 1-1024?  I seem to recall those being reserved ports

Reply:
Ran `make docs`, excluded ports [1,1023]